src/code.cloudfoundry.org/silk/controller/leaser/cidrpool_test.go

```diff
@@ -121,24 +213,33 @@ var _ = Describe("CIDRPool", func() {
 			for result := range results {
 				_, subnet, err := net.ParseCIDR(result)
 				Expect(err).NotTo(HaveOccurred())
-				Expect(network.Contains(subnet.IP)).To(BeTrue())
+				isInNetwork0 := networks[0].Contains(subnet.IP)
+				isInNetwork1 := networks[1].Contains(subnet.IP)
+
+				Expect(isInNetwork0).To(BeTrue())
+				Expect(isInNetwork1).To(BeFalse())
+
 				Expect(subnet.Mask).To(Equal(net.IPMask{255, 255, 255, 255}))
-				// 10.255.0.0 should never be allocated? Maybe?
-				Expect(subnet.IP.To4()).NotTo(Equal(network.IP.To4()))
+
+				//The first IP in the lease should never be allocated
+				Expect(subnet.IP.To4()).NotTo(Equal(networks[0].IP.To4()))
 			}
 		})
 
 		Context("when the subnet mask is 29", func() {
-			It("returns ips containing only .1-.7", func() {
-				subnetRange := "10.255.0.0/16"
-				_, network, _ := net.ParseCIDR(subnetRange)
+			It("returns ips containing only .1-.7 from the first overlay network cidr", func() {
```

Comment:
is this test case really any different from the one before? If it's just here to make sure we don't hardcode stuff related to subnet size when the above test goes for /24, can we change the Context/It message to be more explicit about that?